state/migration/model.go

```diff
@@ -29,13 +30,14 @@ type ModelArgs struct {
 func NewModel(args ModelArgs) Model {
 	m := &model{
 		Version:             1,
-		Owner_:              args.Owner.Canonical(),
+		Owner_:              args.Owner.Id(),
```

Comment:
Because an empty tag does not result in an empty Canonical strings, but "@local", which isn't useful.

I was wanting to test for an empty owner in the validation.